### DISCUSSION TITLE
Winget Package

### DISCUSSION THREAD

**Discussion by timoguin** [FiloSottile/age#469] [General]
I've been trying out winget on Windows 11. It would be nice to have a package for age. I'm not sure what all would be involved.

Are there any plans to create a winget package? Is this something others would be interested in?

**Comment by arebee** [comment]
I did a query tonight and it appears that there is now a winget package for age.

### Search

```
winget search --id FiloSottile.age --verbose
```

### Query Details

```
winget show --id FiloSottile.age
```

### Install 

```
winget install --id FiloSottile.age
```
